What's Added

jitesh-spryiq opened the pull request:
- Added the following new operations and playbooks:
  - Get Alert Responses
  - Get Installed Software
  - Get Endpoints By Search Query

jitesh-rathod committed:
#### What's Added
- Added the following new operations and playbooks:
  - Get Alert Responses
  - Get Installed Software
  - Get Endpoints By Search Query

cs-utkarsh-surana commented on fidelis-edr/info.json:
```diff
@@ -1,1047 +1,1336 @@
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "0",
```
Please add the description in all parameters

cs-utkarsh-surana commented on fidelis-edr/info.json:
```diff
@@ -1,1047 +1,1336 @@
-              "relation": ""
+            "operation": "get_installed_software",
+            "title": "Get Installed Software",
+            "description": "Retrieves information installed software.",
```
Please add more descriptive description.

cs-utkarsh-surana commented on fidelis-edr/playbooks/playbooks.json:
```diff
@@ -1238,161 +1349,80 @@
           "triggerLimit": null,
-          "name": "Get Script Packages",
-          "aliasName": null,
+          "description": "Retrieves information alert responses.",
```
Please add more descriptive description.

jitesh-rathod committed:
A.Added the following new operations and playbooks:
1. Get Alert Responses
2. Get Installed Software
3. Get Endpoints By Search Query
4. Get Job Status By Job ID
5. Create Custom Task

B. Addressed review committee’s feedback by adding more details to action descriptions.

jitesh-rathod committed:
A.Added the following new operations and playbooks:
1. Get Alert Responses
2. Get Installed Software
3. Get Endpoints By Search Query
4. Get Job Status By Job ID
5. Create Custom Task

B. Addressed review committee’s feedback by adding more details to action descriptions.

jitesh-rathod committed:
Added the following new operations and playbooks:
1. Get Alert Responses
2. Get Installed Software
3. Get Endpoints By Search Query
4. Get Job Status By Job ID
5. Create Custom Task

Addressed review committee’s feedback by adding more details to action descriptions.
